javascript-data-model/order-history.js

```diff
@@ -0,0 +1,22 @@
+const orderHistory = [
```

Comment:
Not a bad first attempt, but if you were to scale this out to dozens or hundreds of orders, this may be difficult to work with.

Having an explicit property name with a date doesn't scale well (ie. `deliveredOnJuly52020`). It would be better to have an `orderDate` with a value of a date.

I'll go over the solution in Q&A and describe some of the reasons why this model may not be ideal.
Try and sort each index in your `orderHistory` array as a separate order (one of the rows in the screenshot and find all the common demoninators: 

```js
{
     orderNumber: '12345-23456',
     dateOrdered: '7/19/2020',
     total: 44.35,
     items: [
          {
               name: 'itemName',
               itemTotal: 34.99,
               returnWindow: '8/22/2020'
          }
      ]
 }
```